frontend-web/webclient/app/MainContainer/MainContainer.tsx

```diff
@@ -125,14 +125,14 @@ function SidebarScroll(props: {children: React.ReactNode}): JSX.Element {
         return () => {
             element.onwheel = old;
         }
-    });
+    }, []);
 
-    function scrollMainContainer(e) {
+    function scrollMainContainer(this: GlobalEventHandlers, e: WheelEvent) {
         var elmnt = document.querySelector<HTMLInputElement>(
```

Comment:
It is definitely not an `HTMLInputElement`. I completely forgot to mention this yesterday.